fall-through-cache/fall-through-cache.js

```diff
@@ -147,22 +251,25 @@ module.exports = connect.behavior("fall-through-cache",function(baseConnect){
 			set = set || {};
 			var self = this;
 			return this.cacheConnection.getListData(set).then(function(data){
-
 				// get the list that is going to be made
 				// it might be possible that this never gets called, but not right now
 				self._getHydrateList(set, function(list){
+
+					self._setIsConsistent(list, false);
 					self.addListReference(list, set);
 
 					setTimeout(function(){
 						baseConnect.getListData.call(self, set).then(function(listData){
 
+							self._setInconsistencyReason(list, undefined);
 							self.cacheConnection.updateListData(listData, set);
 							self.updatedList(list, listData, set);
+							self._setIsConsistent(list, true);
 							self.deleteListReference(list, set);
 
 						}, function(e){
-							// what do we do here?  self.rejectedUpdatedList ?
-							console.log("REJECTED", e);
+							console.error("baseConnect.getListData rejected", e);
+							self.rejectedUpdatedInstance(list, e);
```

Comment:
I don't, "instance" was the best neutral name I could come up with to apply to both a map and a list.